cells-erb.gemspec

```diff
@@ -13,6 +13,12 @@ Gem::Specification.new do |spec|
   spec.homepage      = 'https://github.com/trailblazer/cells-erb'
   spec.license       = 'MIT'
 
+  spec.metadata['bug_tracker_uri'] = "#{spec.homepage}/issues"
+  spec.metadata['changelog_uri'] = "#{spec.homepage}/blob/HEAD/CHANGES.md"
+  spec.metadata['documentation_uri'] = "https://www.rubydoc.info/gems/cells-erb/#{spec.version}"
+  spec.metadata['homepage_uri'] = spec.homepage
+  spec.metadata['source_code_uri'] = spec.homepage
```

Comment:
@apotonick do these look right? Should the `documentation_uri` point to https://trailblazer.to/2.1/docs/cells like the `cells` gem?

Reply:
I think this is fine! Until we have specific docs for ERBse we can point to the Cells gem's docs.